contracts/Core/Parameters.sol

```diff
@@ -112,12 +113,80 @@ contract Parameters is ACL, Constants {
         escapeHatchEnabled = false;
     }
 
-    function getEpoch() external view returns (uint32) {
+    function getEpoch() external view override returns (uint32) {
         return (uint32(block.number) / (epochLength));
     }
 
-    function getState() external view returns (uint8) {
+    function getState() external view override returns (uint8) {
         uint8 state = uint8(((block.number) / (epochLength / NUM_STATES)) % (NUM_STATES));
         return (state);
     }
+
+    function getEpochLength() external view override returns (uint16) {
```

Comment:
hey is there a reason to add these getters ?

Reply:
I have removed them.